test/test.cpp

```diff
@@ -0,0 +1,6 @@
+#include "test.h"
+
+GTEST_API_ int main(int argc, char **argv) {
```

Comment:
Why not just link with gtest_main?

Reply:
Just to have a potential place where you could add additional test code or test engines which doesn't fit in googletest. Also to get rid of useless startup printf 'Running main() from ...'

Reply:
OK makes sense, can place global initialisation here.